### PR TITLE
Corrections for CD

### LINKED ISSUE #1015: Further changes required for CD
When running through the release process problems arose that require changes to the codebase.  We will need to fix these problems for the release process to work smoothly.

### PR DESCRIPTION
**Pull request by hsorby**
Fixes #1015.

### CODE REVIEW

File: cmake/add_rpath_for_python.cmake [hunk 1]
```diff
@@ -2,6 +2,7 @@
 # Make the SWIG generated Python interface modules more portable by using rpath for the the Python dynamic library.
 string(REPLACE "${LIST_SEPARATOR}" ";" SHARED_OBJECT_MODULES "${SHARED_OBJECT_MODULES}")
 foreach(_M ${SHARED_OBJECT_MODULES})
-  execute_process(COMMAND ${INSTALL_NAME_TOOL_EXE} -change "/Library/Frameworks/Python.framework/Versions/${Python_VERSION_MAJOR}.${Python_VERSION_MINOR}/Python" @rpath/${Python_VERSION_MAJOR}.${Python_VERSION_MINOR}/Python ${_M})
+  execute_process(COMMAND ${INSTALL_NAME_TOOL_EXE} -change "/Library/Frameworks/Python.framework/Versions/${Python_VERSION_MAJOR}.${Python_VERSION_MINOR}/Python" @rpath/Python.framework/Versions/${Python_VERSION_MAJOR}.${Python_VERSION_MINOR}/Python ${_M})
+  execute_process(COMMAND ${INSTALL_NAME_TOOL_EXE} -add_rpath /opt/homebrew/Frameworks ${_M})
```
Comment (agarny) [inline cmake/add_rpath_for_python.cmake:6]: Why do we need this? Not to mention that we can't assume that people will have Homebrew installed on their Mac?

**Review by agarny — COMMENTED** [review]
Looks fine except for my Homebrew-related comment.

File: cmake/add_rpath_for_python.cmake (hunk 1 above)
Comment (hsorby) [reply cmake/add_rpath_for_python.cmake:6]: We don't **need** it simply adds another path to search for the Python framework.  If you don't have a path then no issue.

Comment (hsorby) [reply cmake/add_rpath_for_python.cmake:6]: It is intended to increase portability.

Comment (agarny) [reply cmake/add_rpath_for_python.cmake:6]: Yes, I eventually gathered that. All good then.

**Review by agarny — APPROVED** [review]

**Review by nickerso — APPROVED** [review]

**Pull request merged**